plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -16,6 +17,7 @@ type SQLServer struct {
 	QueryVersion  int      `toml:"query_version"`
 	AzureDB       bool     `toml:"azuredb"`
 	ExcludeQuery  []string `toml:"exclude_query"`
+	QueryTimeout  int      `toml:"query_timeout"`
```

Comment:
Use `internal.Duration` for this option, so that the timeout can be specified using the `"5s"` format.